### PR TITLE
git-clear: add page

### PR DESCRIPTION
**Pull request by Flodur871**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).
 
Added `git clear` command from `tj/git-extras` as mentioned in #5137

### CODE REVIEW

File: pages/common/git-clear.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# git clear
+
+> Hard reset git branch, deletes all untracked files including the ones in .gitignore.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-brv>.
```
Comment (navarroaxel) [inline pages/common/git-clear.md:5]:
```suggestion
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-clear>.
```

File: pages/common/git-clear.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# git clear
+
+> Hard reset git branch, deletes all untracked files including the ones in .gitignore.
```
Comment (navarroaxel) [inline pages/common/git-clear.md:3]:
```suggestion
> Clear a Git working directory as if it was freshly cloned with the current branch.
```

File: pages/common/git-clear.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-brv>.
+
+- Hard reset git branch, deletes all untracked files including the ones in .gitignore:
```
Comment (navarroaxel) [inline pages/common/git-clear.md:7]:
```suggestion
- Reset all tracked files, and delete all untracked files even if they are included in the `.gitignore`:
```

File: pages/common/git-clear.md (hunk 1 above)
Comment (Flodur871) [reply pages/common/git-clear.md:5]: Got me

**Review by navarroaxel — APPROVED** [review]

File: pages/common/git-clear.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# git clear
+
+> Clear a Git working directory as if it was freshly cloned with the current branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-clear>.
+
+- Reset all tracked files, and delete all untracked files even if they are included in the `.gitignore`:
```
Comment (CleanMachine1) [inline pages/common/git-clear.md:7]:
```suggestion
- Reset all tracked files and delete all untracked files even if they are included in the `.gitignore`:
```

File: pages/common/git-clear.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# git clear
+
+> Clear a Git working directory as if it was freshly cloned with the current branch.
```
Comment (CleanMachine1) [inline pages/common/git-clear.md:3]:
```suggestion
> Clear a Git working directory as if it was freshly cloned with the current branch including files in `.gitignore`.
```

**Review by sbrl — APPROVED** [review]
Thanks, @Flodur871!

**Pull request merged**